serverless.js

```diff
@@ -96,6 +77,48 @@ module.exports = {
   },
   resources: {
     Resources: {
+      HttpApiDomain: {
+        Type: 'AWS::ApiGatewayV2::DomainName',
+        Properties: {
+          DomainName: `api.\${self:custom.origin}`,
```

Comment:
This means it's no longer on the same domain as the frontend? That seems non-ideal as it will require us to take care of the corresponding CORS settings, right?

Reply:
Life is made of decisions 🙂. We can choose to have a record set pointing to CloudFront or the HttpApiGateway.

Reply:
How does it work for the branch deploys? `api.my-branch.asap.yld.io`?

Reply:
Yeah, apparently

Reply:
We can ship this as is for now I guess?
Two questions:
This means things are no longer deployed to a cloudfront domain, ONLY the `*.asap.yld.io`, correct?
And once the time comes to connect FE and API, would you rather configure CORS rules or try to get them hosted on one origin again?

Reply:
CORS rules is my bet.